### DISCUSSION TITLE
Feature request:  User selectable colour for Draw Grid

### DISCUSSION THREAD

**Discussion by IronOfIron** [IENT/YUView#467] [Ideas]
In working with YUV files (sometimes almost all white/black), I typically put a grid on it (16x16, 64x64, etc), using the Draw Grid feature.  Since this grid is coloured in black, I cannot see it when my image is dark.  It would be very useful if there could be an option to select the colour of the grid so that you could see it in any sceanrio.  Thanks!

**Comment by ChristianFeldmann** [comment]
Thanks for the idea. I created a feature request issue: https://github.com/IENT/YUView/issues/473

**Comment by IronOfIron** [comment]
Great thanks!

**Comment by ChristianFeldmann** [comment]
Just as an update to this. This is already possible. In the settings, the grid color can be customized.

**Comment by IronOfIron** [comment]
Wow I totally missed that.  Thanks for pointing that out.
